config/config-docker.yaml

```diff
@@ -0,0 +1,31 @@
+service:
```

Comment:
Do we need the config for docker or will overriding the environmental variables be good enough?

Reply:
Unfortunately this service is not 12FA and doesn't get configured via it's environment. Instead a YAML file is used to configure the service as per https://github.com/ONSdigital/ras-collection-instrument/blob/20cce0c57516e373adf4d3b67321f40e696a717a/app.py#L14

Reply:
I will raise a tech debt issue here and on trello to get this app configured via it's environment.

Reply:
I think it has some support around env variables but don't know how it works fully https://github.com/ONSdigital/ras-common-utils/blob/master/ras_common_utils/ras_config/ras_config.py#L38

Reply:
I was unable to configure this application from the environment so what I did was to change run.py to allow an env variable to specify the config file so you could override the config from the environment.  I ran this approach past a couple of people and they were happy it was an improvment on what went before :-) I can't see what code changes are required here (short of rewriting the whole configuration piece, which is beyond the scope of this card)